testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/ContactListAdapter.kt

```diff
@@ -19,18 +21,30 @@ class ContactListAdapter : RecyclerView.Adapter<ContactListAdapter.ViewHolder?>(
     }
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
-        holder.contactName.text = contactEntries[position]
+        val contact = contactEntries[position]
+        holder.contactId.text = contact.id
```

Comment:
Then, maybe we also can write: `bindNonVisibleView("Id: ", contact.id, holder.contactId)` ?